src/login/logind-session.c

```diff
@@ -656,6 +657,14 @@ static int session_start_scope(Session *s, sd_bus_message *properties, sd_bus_er
 
                 description = strjoina("Session ", s->id, " of User ", s->user->user_record->user_name);
 
+                after = strv_new("systemd-logind.service",
+                                 s->user->runtime_dir_service,
+                                 s->user->service,
+                                 s->user->user_record->uid > 0 ? "systemd-user-sessions.service" : STRV_IGNORE,
+                                 s->class != SESSION_BACKGROUND ? s->user->service : STRV_IGNORE);
```

Comment:
hmm, what? you order after `s->user->service` twice now. here and two lines up...

Reply:
Not addressed.